### ISSUE TITLE
Received new version 1.9.1523.0 from Windows Store - Can find no matching tag/build in repo, nor change log

### ISSUE THREAD

**Issue by BouwenMA** [microsoft/terminal#10314]
Already had v1.9.1445.0 updated several days ago, and just today had update come down from store for a version 1.9.1523.0.  The "About" screen shows the newer 1.9.1523.0 version, the recent activity in the store for downloads and updates also shows the new 1.9.1523 version, yet the store page itself still reports it as v1.9.1445.0.

It is not readily apparent where this build came from given there are no matching tags/releases in the github repo.

**Comment by skyline75489** [comment]
Yeah sometimes the store release comes before the tags/releases in the Github repo.

> where this build came from

Well, if you really want to know the answer, the build actually came from Microsoft *internally*. The Github repo does not contain all the needed things. There's a bunch of things needed inside Microsoft (for example, signing, localization, packaging, etc.) to publish a new release.

**Comment by DHowett** [comment]
Yes, sorry. I usually only tag the release after it's up on the store.

The delta you'll be interested in is `v1.9.1444.0..release-1.9`. Everything's there in the commit history for the 1.9 branch.

**Comment by DHowett** [comment]
Tags pushed; getting the release notes up.

https://github.com/microsoft/terminal/tree/v1.9.1523.0
https://github.com/microsoft/terminal/tree/v1.8.1521.0

**Comment by DHowett** [comment]
Sorry about the miscommunication! The releases actually are built totally out of this repository :)

There are a couple overlay files but this is the only repository that we use.

**Comment by vefatica** [comment]
When will these be available on GitHub?
